webapp/src/vue-apollo.js

```diff
@@ -0,0 +1,78 @@
+import Vue from "vue";
```

Comment:
Awesome you implemented the frontend-side as well! :clap: :smiley: 
Since I didn't dive into this on my own yet, I can't go review this in detail, though, sorry! :cry: 

However, I see one big issue: You seem to have copied this file in whole from this [stackoverflow answer](https://stackoverflow.com/a/61239384).
The copied code is definitely original work and since it seems to originate from the linked answer, as far as I can tell it's licensed under [CC-BY-SA](https://creativecommons.org/licenses/by-sa/3.0/) - which requires you to give reference to the original author and to license your derived work under the same (or compatible) license.
Both requirements are not met as far as i can see.

Apart from the legal issues (which don't really have practical implications to be honest), copying stuff without attribution is kind of questionable in academia :stuck_out_tongue:

Reply:
I didn't copy the file from anywhere.  It was generated by calling 
`vue add apollo` which I used to quickly setup apollo-client.

Reply:
Oh wow, I see! Then it's not an issue at all, of course :smile: